src/healthcareapis/azext_healthcareapis/azext_metadata.json

```
@@ -1,3 +1,3 @@
 {
-    "azext.minCliCoreVersion": "2.15.0"
+    "azext.minCliCoreVersion": "2.61.0"
```

The identity should be used in version above 2.66.0
```suggestion
    "azext.minCliCoreVersion": "2.66.0"
```

---

Done